setup.py

```diff
@@ -14,8 +14,9 @@
 extras = {
     "hdf5": ["h5py"],
     "zarr": ["zarr"],
-    "vigra": ["vigra"],
-    "nifty": ["nifty"]
+    "vigra": ["vigra"],  # conda-only
+    "nifty": ["nifty"],  # conda-only; name clash w/ different PyPI package
+    # "n5": ["pyn5"],  # PyPI-only
```

Comment:
I think we should restructure the extra dependencies a bit, see my comment in #2.